plutus-tx/src/PlutusTx/Lift/TH.hs

```diff
@@ -145,7 +145,7 @@ In the Lift case, type variables will be free type variables in the instance, an
 by appropriate Typeable constraints for those variables. We get the PLC types by just calling
 typeRep.
 
-However, for simplicity we always act as though we have a local scope, and fall back to Typeable,
+However, for simplicity we always act as though we have a local scope, and fall back to
```

Comment:
this sentence needs to be updated

Reply:
Got lazy and did it with command line foo. Will fix.